### PR TITLE
Correct nodejs spec filenames

### PR DESCRIPTION
**Pull request by ekohl**

### CODE REVIEW

File: nodejs-brace/nodejs-brace.spec [hunk 1]
```diff
@@ -22,8 +22,8 @@ ExclusiveArch: %{nodejs_arches} noarch
 ExclusiveArch: %{ix86} x86_64 %{arm} noarch
 %endif
 Provides: npm(%{npm_name}) = %{version}
-Provides: bundled-npm(brace) = 0.10.0
-Provides: bundled-npm(w3c-blob) = 0.0.1
+Provides: bundled(npm(brace)) = 0.10.0
```
Comment (dLobatog) [inline nodejs-brace/nodejs-brace.spec]: Why is this change needed here and not in other specs?

Comment (ekohl) [reply nodejs-brace/nodejs-brace.spec]: @xsuchy you added this in your PR. Can you elaborate? Is this something we should be doing in all our specs?

Comment (xsuchy) [reply nodejs-brace/nodejs-brace.spec]: @ekohl  Yes. It should be everywhere. See https://fedoraproject.org/wiki/Packaging:Guidelines#Bundling_and_Duplication_of_system_libraries

Comment (ekohl) [reply nodejs-brace/nodejs-brace.spec]: In that case I'll rewrite this PR to only rename the spec files to remove the bundled and so the bulk refactor in another PR.
